File: src/main/java/net/glasslauncher/mods/glassguis/events/init/GlassGUIs.java
package net.glasslauncher.mods.glassguis.events.init;

import com.github.benmanes.caffeine.cache.Cache;
import com.github.benmanes.caffeine.cache.Caffeine;
import org.apache.logging.log4j.Level;
import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;
import org.apache.logging.log4j.core.config.Configurator;
import org.jetbrains.annotations.NotNull;

public class GlassGUIs {
    public static final Cache<String, int[]> IMAGE_SIZE_CACHE = Caffeine.newBuilder().softValues().build();
//    public static @NotNull Logger LOGGER;
//
//    static  {
//        LOGGER = LogManager.getLogger("GlassGUIs");
//        Configurator.setLevel(LOGGER, Level.INFO);
//    }
}

File: src/main/kotlin/net/glasslauncher/mods/glassguis/events/init/GlassGUIs.kt
package net.glasslauncher.mods.glassguis.events.init

import net.modificationstation.stationapi.api.mod.entrypoint.EntrypointManager
import net.modificationstation.stationapi.api.util.Namespace
import org.apache.logging.log4j.Logger
import java.lang.invoke.MethodHandles

class GlassGUIs {
    companion object {
        @Suppress("UnstableApiUsage")
        val NAMESPACE: Namespace = Namespace.resolve()
        val LOGGER: Logger = NAMESPACE.getLogger("GlassGUIs")

        init {
            EntrypointManager.registerLookup(MethodHandles.lookup())
        }
    }
}